Readme.md

```diff
@@ -4,19 +4,18 @@
 
 | Windows (x86, x64) |
 |---------|
-|![![Build status](https://ci.appveyor.com/api/projects/status/67mildjynhnlekk5/branch/master?svg=true)](https://ci.appveyor.com/project/Raffaello/sdl2-vga-terminal/branch/master)|
+|[![Build status](https://ci.appveyor.com/api/projects/status/67mildjynhnlekk5/branch/master?svg=true)](https://ci.appveyor.com/project/Raffaello/sdl2-vga-terminal/branch/master)|
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[table-cell-padding] Cell should be padded](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5369200)